address/book.go

```diff
@@ -193,15 +194,70 @@ func NewBook(cfg BookConfig) *Book {
 	}
 }
 
+// queryAssetInfo attempts to locate asset genesis information by querying
+// geneses already known to this node. If asset issuance was not previously
+// verified, we then query universes in our federation for issuance proofs.
+func (b *Book) queryAssetInfo(ctx context.Context,
+	id asset.ID) (*asset.AssetGroup, error) {
+
+	// Check if we know of this asset ID already.
+	assetGroup, err := b.cfg.Store.QueryAssetGroup(ctx, id)
+	switch {
+	case assetGroup != nil:
+		return assetGroup, nil
+
+	// Asset lookup failed gracefully; continue to asset lookup using the
+	// AssetSyncer if enabled.
+	case errors.Is(err, ErrAssetGroupUnknown):
+		if !b.cfg.SyncerActive {
+			return nil, ErrAssetGroupUnknown
+		}
+
+	case err != nil:
+		return nil, err
+	}
+
+	log.Debugf("asset %v is unknown, attempting to bootstrap", id.String())
+
+	// Use the AssetSyncer to query our universe federation for the asset.
+	err = b.cfg.Syncer.FetchAssetInfo(ctx, &id)
```

Comment:
Ah I see here we use sync vs fetch somewhat interchangeably. Perhaps use a `Sync` prefix here since we're reaching out to a remove server to try to obtain the information?